config.toml

```diff
@@ -33,3 +33,13 @@ weight = -100
 name = "会议日程"
 url = "zh/tracks.html"
 weight = -100
+
```

Comment:
I don't think we need to add the manual here, we can add the sessions link into the track page.